tutorials/2_Gazebo_and_ROS.ipynb

```diff
@@ -0,0 +1,1330 @@
+    "<ul>\n",
+    "<li>Where was the head frame relative to the world frame, 5 seconds ago?</li>\n",
+    "<li>What is the pose of the object in my gripper relative to my base?</li>\n",
+    "<li>What is the current pose of the base frame in the map frame?</li>\n",
+    "</ul>\n",
+    "\n",
+    "<p>You can get started on the introduction to tf tutorial. For a complete list of all tf and tf-related <a href=\"http://wiki.ros.org/tf/Tutorials\">tutorials</a> check out the tutorials page.</p>\n",
+    "\n",
+    "\n",
+    "<h2>Universal Robotic Description Format (URDF)</h2>\n",
+    "<p>The Universal Robotic Description Format (URDF) is an XML file format used in ROS to describe all elements of a robot. To use a URDF file in Gazebo, some additional simulation-specific tags must be added to work properly with Gazebo. This tutorial explains the necessary steps to successfully use your URDF-based robot in Gazebo, saving you from having to create a separate Simulation Description Format (SDF) file from scratch and duplicating description formats. Under the hood, Gazebo will then convert the URDF to SDF automatically. Xacro (XML Macros) is an XML macro language. It is very useful to make shorter and clearer robot descriptions.</p>\n",
+    "\n",
+    "<h2>URDF Tags</h2>\n",
+    "<p>Main tags we will use in writing our URDF are as follows:</p>\n",
+    "<ul>\n",
+    "<li><strong><a href=\"http://wiki.ros.org/urdf/XML/link\">link tag:</a></strong> Describes the kinematic and dynamic properties of a link.</li>\n",
+    "<li><strong><a href=\"http://wiki.ros.org/urdf/XML/Transmission\">transmission tag:</a></strong> Transmissions link actuators to joints and represents their mechanical coupling</li>\n",
+    "<li><strong><a href=\"http://wiki.ros.org/urdf/XML/joint\">joint tag:</a></strong> Describes the kinematic and dynamic properties of a joint.</li>\n",
+    "<li><strong><a href=\"http://wiki.ros.org/urdf/XML/Gazebo\">gazebo tag:</a></strong> Describes simulation properties, such as damping, friction, etc</li>\n",
+    "<li><strong><a href=\"http://wiki.ros.org/urdf/XML/sensor\">sensor tag:</a></strong> Describes a sensor, such as a camera, ray sensor, etc</li>\n",
+    "</ul>\n",
+    "\n",
+    "<h2>Simulate differential drive mobile robot in Gazebo and ROS</h2>\n",
```

Comment:
Simulat**ing** a differential drive mobile robot in Gazebo and ROS

The sections after this one should all be subsections (lower heading number).